common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -200,7 +200,9 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
 #}
 #
   @markdownToXml: (markdown)->
-    toXml = `function (markdown) {
+    `// Translators: This is the title of the text explaining a problem`
+    explanation_title = gettext("Explanation")
+    toXml = `function (markdown, explanation_title) {
```

Comment:
I suspect that changing the function signature here is causing issues. Can you do the translation within this function, ie perhaps the line above the `var selectString` is defined?